apps/02RockPaperScissors.js

```diff
@@ -10,7 +10,30 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+if (hand1 === hand2) {
+   return 'It\'s a tie!';
+  }
+
+if (hand1 === 'rock') {
```

Comment:
add .toLowercase() to the end of 'hand1' and 'hand2' to potentially deter a user error